src/lib.rs

```diff
@@ -217,26 +220,31 @@ impl Surreal {
     #[napi]
     pub async fn create(&self, resource: String, data: Option<Value>) -> Result<Value> {
         let resource = Resource::from(resource);
-
-        let response = match data {
-            Some(d) => self.db.create(resource).content(d).await.map_err(err_map)?,
+		let response = match data {
             None => self.db.create(resource).await.map_err(err_map)?,
+            Some(d) => {
+                let d = json(&d.to_string()).unwrap();
```

Comment:
I know it's unlikely to run into an error here but it's possible. Kindly propagate it please.